tasks/electron.js

```diff
@@ -73,7 +73,8 @@ function _processZipEntry(zip, dest, entry) {
   }
 }
 
-function _zipExtract(zipfile, dest = './') {
+function _zipExtract(zipfile, dest) {
+  dest = (typeof dest !== 'undefined') ? dest : './';
```

Comment:
Yes, on Debian the supported node is still 4, based on an internet search the same for Ubuntu :(

I didn't have to modify any other part of the code, and I'm sure you are using new ES syntax elsewhere too (I agree btw, they are very convenient and nice!)  So maybe the babel setup is not working for this file, I don't know.

If you want, I can run "npm install" without this and send you the error message if you are not willing to merge this.  Why I would like to get this merged, because it's already a git checkout and npm install + npm run dev for the linux guys, and telling them to install node 6, when their distribution doesn't have it seems complicated.

Thanks for the consideration!